build.gradle

```diff
@@ -122,6 +128,17 @@ def signingKey(project) {
       }
     } // signingConfigs
 
+    playAccountConfigs {
+      defaultAccountConfig {
+        serviceAccountEmail = 'name@server.com'
+        pk12File = project.file('key.p12') // Will need to encrypt this in Travis, obvs
```

Comment:
Yes, I get this by the way:

![screen shot 2017-06-06 at 00 13 45](https://cloud.githubusercontent.com/assets/361423/26807215/0f1fe250-4a4d-11e7-84ea-6704062995f3.png)